data/software_stack/eessi.yaml

```diff
@@ -0,0 +1,47 @@
+jupyterhub::kernel::setup: venv
+jupyterhub::kernel::venv::python: "/cvmfs/pilot.eessi-hpc.org/2020.10/compat/%{facts.os.architecture}/usr/bin/python3"
```

Comment:
This is portable but not optimal...it is using the Gentoo prefix python (which does have venv). It also assumes that `%{facts.os.architecture}` is the same as the output of `uname -m` which seems to be the case, but could bite us on other archs.

I can't think of another way right now...

Reply:
This doesn't work as intended because the Gentoo python is not configured properly to pick up python packages installed with EasyBuild